### PR TITLE
Add the first section of the KT TTT Tutorial Text.

### PR DESCRIPTION
**Pull request by SHeimlich**
This is just the first section. Just to make review easier, and get things in as they get written.

### CODE REVIEW

File: particles/Tutorial/Kotlin/Demo/README.md [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+where particles are made by external developers and combined to form recipes - much the way 
+developers use external libraries in object-oriented languages.
+
+However, making particles as small as possible leads us towards a "turtles all the way down"
+mentality. There is obvious overhead involved in each particle, therefore the desire for small
+particles must be balanced against having too many.
+
+#### Bounds of Data Knowledge
+Arcs' mission statement says that users have sovereignty over their data and its use. To ensure
+data only travels as intended, Arcs uses data flow analysis to make grantees about data flow. But,
```
Comment (csilvestrini) [inline particles/Tutorial/Kotlin/Demo/README.md]: grantees -> guarantees

File: particles/Tutorial/Kotlin/Demo/README.md [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+for this to work properly, particles must delineate knowledge boundaries. Just as each nation has a 
+border so customs and immigration can know where to operate, we need to create systems with well 
+defined boundaries so Arcs can know where data should (and more importantly, shouldn't) exist.
```
Comment (csilvestrini) [inline particles/Tutorial/Kotlin/Demo/README.md]: exist -> go

File: particles/Tutorial/Kotlin/Demo/README.md [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+
+
+#### Functional Components
+When we create recipes, there are going to be occasions when it makes sense to extract some 
+functionality its own particle. This is easiest to understand with an example, which we provide in
```
Comment (csilvestrini) [inline particles/Tutorial/Kotlin/Demo/README.md]: its -> into its

File: particles/Tutorial/Kotlin/Demo/README.md [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+more detail below.
+
+### Tic-Tac-Toe, here we go!
+
+Alright, so let's get to putting this in practice with our tic-tac-toe game. As with any good
+engineering project, we need a set of requirements. Here are ours:
+ 1. Play Tic Tac Toe with the traditional rules.
+ 2. Be able to have a human play against a computer.
+ 3. The players (human or computer) may make invalid moves at invalid times.
+ 4. The main UI should not be trusted with any personal information, such as names.
```
Comment (csilvestrini) [inline particles/Tutorial/Kotlin/Demo/README.md]: This one seems a little odd (doesn't the UI need to display the name?).

Could you rephrase it a little? Something like "Personal information, such as names, should be restricted to the components that need to know about them"

File: particles/Tutorial/Kotlin/Demo/README.md [hunk 5]
```diff
@@ -0,0 +1,89 @@
...
+
+However, making particles as small as possible leads us towards a "turtles all the way down"
+mentality. There is obvious overhead involved in each particle, therefore the desire for small
+particles must be balanced against having too many.
+
+#### Bounds of Data Knowledge
+Arcs' mission statement says that users have sovereignty over their data and its use. To ensure
+data only travels as intended, Arcs uses data flow analysis to make grantees about data flow. But,
+for this to work properly, particles must delineate knowledge boundaries. Just as each nation has a 
+border so customs and immigration can know where to operate, we need to create systems with well
```
Comment (Cypher1) [inline particles/Tutorial/Kotlin/Demo/README.md]: I like the analogy used here, but I wonder if an immune system or just a company, made up of teams, might be less political? Just an idea rather than a requested change

File: particles/Tutorial/Kotlin/Demo/README.md (hunk 4 above)
Comment (Cypher1) [reply particles/Tutorial/Kotlin/Demo/README.md]: Presumably this is for an anonymous game of tic tac toe (the stakes may be high and we don't want people to hold a grudge :P )

Comment (SHeimlich) [reply particles/Tutorial/Kotlin/Demo/README.md]: The use case @shaper and I joke about is that maybe you're playing during work hours, so you don't want your boss to know....
